index.html

```diff
@@ -40,24 +38,24 @@ <h1>Startrly</h1>
       </section>
 
 
-      <section>
+      <section class="why">
         <h2>Why?</h2>
-        <ul>
-          <li>
+        <ul class="answers">
+          <li class="answerbox">
```

Comment:
Instead of giving the li elements `class` attributes, could you leave them as-is and just refer to them in the css as belonging to their parent `ul`? Like `.answers li` would cover all the list items inside the `.answers` ul.